test/powershell/Modules/Microsoft.PowerShell.Security/AclCmdlets.Tests.ps1

```diff
@@ -3,18 +3,18 @@
 Describe "Acl cmdlets are available and operate properly" -Tag CI {
     It "Get-Acl returns an ACL object" -pending:(!$IsWindows) {
         $ACL = get-acl $TESTDRIVE
-        $ACL | Should BeOfType "System.Security.AccessControl.DirectorySecurity"
+        $ACL | Should -BeOfType "System.Security.AccessControl.DirectorySecurity"
     }
     It "Set-Acl can set the ACL of a directory" -pending {
         Setup -d testdir
         $directory = "$TESTDRIVE/testdir"
         $acl = get-acl $directory
         $accessRule = [System.Security.AccessControl.FileSystemAccessRule]::New("Everyone","FullControl","ContainerInherit,ObjectInherit","None","Allow")
         $acl.AddAccessRule($accessRule)
-        { $acl | Set-Acl $directory } | should not throw
+        $acl | Set-Acl $directory
```

Comment:
I'd expect `{ $acl | Set-Acl $directory } | Should -Not -Throw`

Reply:
I'll revert it, but I believe that Pester now discourages the use of `$scriptBlock | Should -Not -Throw` because all lines should not throw.